### DISCUSSION TITLE
Add Rewards for Completing an Achievement

### DISCUSSION THREAD

**Discussion by ichlaffterlalu** [DiscordDungeons/Bugs#976] [Ideas]
To draw players' attention on achievement, maybe the game can offer some rewards when a player completing an achievement. Right now we didn't get any rewards when completing an achievement.
Maybe the proper example of "achievement rewards" are Minecraft's advancement rewards.
Rewards can be anything, just like quest rewards: gold, XP, or any memorial item.

**Comment by Feathers8** [comment]
A few rubies for completing those dungeon achievements would be a nice gesture

**Reply by oni-main** [reply]
Ur goddamn right

**Comment by Sven65** [comment]
We'll be adding rewards to achievements, most likely in the form of XP and such.
